projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/components/search-language-interpreter/search-language-interpreter.component.spec.ts

```diff
@@ -241,6 +241,11 @@ describe('SearchLanguageInterpreterComponent', () => {
   it('should show an error message on clicking "Next" if no language has been selected', () => {
     spyOn(component, 'onNext').and.callThrough();
     spyOn(component.caseFlagStateEmitter, 'emit');
+    component.selectedLanguage = <any>{
```

Comment:
we should try avoiding using the any keyword. maybe use {key: string, value: string, value_cy: string}

Reply:
@sonny-hmcts - It was difficult to define the MatOption and set the value. I have now used a different approach that was used in another test.